### PR TITLE
Test and merge

### PR DESCRIPTION
**Pull request by Rednaxela700**

### CODE REVIEW

File: src/components/Loader.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Loader from "react-loader-spinner";
+
+
+export const ShowLoader = () => {
+    return (
+        <div style={{display: 'flex', justifyContent: 'center', alignItems: 'center', width: '100%'}}>
+            <Loader
+                type="TailSpin"
```
Comment (michal-kusnierz) [inline src/components/Loader.js]: nice work

File: src/components/Navbar.js [hunk 2]
```diff
@@ -1,61 +1,68 @@
...
+                    menuItem('Oferta', '/search', 'search')
+                }
+                {
+                    menuItem('Dodaj', '/form/', 'add')
+                }
+                {
+                    menuItem('Ulubione', '/favs', 'heart')
+                }
+                {
+                    menuItem('Panel', '/panel', 'user')
```
Comment (michal-kusnierz) [inline src/components/Navbar.js]: what do we need Panel component for (on master)?

Comment (Rednaxela700) [reply src/components/Navbar.js]: Right, we could hide it on master and restore again on dev. Can you comment it out and conclude merge? I won’t have access to the computer for next 20 hours

Comment (michal-kusnierz) [reply src/components/Navbar.js]: Yup

**Pull request merged**